app/experimenter/experiments/tests/factories/nimbus.py

```diff
@@ -37,10 +38,10 @@ class NimbusExperimentFactory(factory.django.DjangoModelFactory):
         lambda o: decimal.Decimal(random.randint(1, 10) * 10)
     )
     firefox_min_version = factory.LazyAttribute(
-        lambda o: random.choice(NimbusExperiment.Version.choices)
+        lambda o: random.choice(NimbusExperiment.Version.choices)[0]
```

Comment:
These aren't named tuples? Might be useful to use named tuples instead so we can have more meaningful access to the attribute.

Reply:
Ah in theory, but using the Django TextChoices enum converts them to unnamed tuples when you access its choices list

Reply:
Oh! Actually you can treat the enum as an iterable and then you get the nice accessors, let's use those 👍